src/ocm/components/clusterWizard/ClusterWizardContextProvider.tsx

```diff
@@ -14,15 +13,59 @@ import { StaticIpView } from '../clusterConfiguration/staticIp/data/dataTypes';
 import { getStaticIpInfo } from '../clusterConfiguration/staticIp/data/fromInfraEnv';
 import { Cluster, InfraEnv, AssistedInstallerOCMPermissionTypesListType } from '../../../common';
 import useSetClusterPermissions from '../../hooks/useSetClusterPermissions';
+import useClusterCustomManifests from '../../hooks/useClusterCustomManifests';
+import { ListManifestsExtended } from '../clusterConfiguration/manifestsConfiguration/data/dataTypes';
 
-const getWizardStepIds = (staticIpView?: StaticIpView): ClusterWizardStepsType[] => {
-  const stepIds: ClusterWizardStepsType[] = [...defaultWizardSteps];
+const addStepToClusterWizard = (
+  wizardStepIds: ClusterWizardStepsType[],
+  position: number,
+  itemsToAdd: ClusterWizardStepsType[],
+) => {
+  const found = wizardStepIds.filter((step) => step === itemsToAdd[0]);
+  if (found.length === 0) {
+    wizardStepIds.splice(position, 0, ...itemsToAdd);
+  }
+};
+
+const removeStepFromClusterWizard = (
```

Comment:
These functions are much easier to understand IMO.

For the "remove" one, I would simplify it as we don't care in which position the step is at. It's redundant and it can change if new steps are introduced.

Check that I didn't do a mistake, but I think it would be:

```
const removeStepFromClusterWizard = (
  wizardStepIds: ClusterWizardStepsType[],
  itemToRemove: ClusterWizardStepsType,
  numberItemsToRemove: number,
) => {
  const position = wizardStepIds.findIndex((step) => step === itemToRemove);
  if (position !== -1) {
    wizardStepIds.splice(position, numberItemsToRemove);
  }
};
```